### PR TITLE
Add missing api services

### PR DESCRIPTION
**Pull request by alcaeus**
This pull request adds a bunch of missing API services. It also fixes a typo in the transfer API service name, deprecating the old service name properly for removal in 2.0.

### CODE REVIEW

**Comment by alcaeus** [comment]
Regarding the build failure:
> /home/travis/build/TeknooSoftware/mango-pay-bundle/src/Teknoo/MangoPayBundle/Service/MangoApiService.php:39	The class MangoApiService has a coupling between objects value of 21. Consider to reduce the number of dependencies under 13.

Nothing I'd fix for now.

File: src/Teknoo/MangoPayBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -55,11 +66,15 @@ services:
   mangopay.sdk.api_pay_outs.service:
     class: "%mangopay.sdk.api_pay_outs.class%"
     factory: ['@teknoo.mangopaybundle.service.mango_api', 'getApiPayOuts']
-
+    
   mangopay.sdk.api_transfert.service:
     class: "%mangopay.sdk.api_transfert.class%"
     factory: ['@teknoo.mangopaybundle.service.mango_api', 'getApiTransferts']
 
+  mangopay.sdk.api_transfert.service:
```
Comment (alcaeus) [inline src/Teknoo/MangoPayBundle/Resources/config/services.yml]: FYI, there's a duplicated service name after the merge commit

**Comment by alcaeus** [comment]
@frenchcomp I can rebase the branch and resolve conflicts if you'd like me to...

**Comment by frenchcomp** [comment]
About the issue, PHPMd is a static detector, in this case, we can disable the test about couplings severals objects by adding "@suppresswarnings(PHPMD.CouplingBetweenObjects)" in the phpdoc of the classes. 
It's not an issue for "MangoApiService" because is a "hub" to prepare and get other MangoPay services. 
@alcaeus : You can rebase, I must reinstall the project on my pc, i use the online editor...

**Comment by alcaeus** [comment]
@frenchcomp Rebase done and issues fixed! 👍

**Comment by alcaeus** [comment]
@frenchcomp could you tag 1.1 now that these are released? Thanks!

**Comment by frenchcomp** [comment]
Hi @alcaeus , Sadly, an issue has been spotted with PHP 7.0 on my CI server.
I will check it this tonight,  I work on a release on another project for one my client.

**Comment by alcaeus** [comment]
No problem, let me know if I can help 👍

**Comment by alcaeus** [comment]
@frenchcomp any updates on the new release?

**Comment by frenchcomp** [comment]
Ho
It cas released the last week :)

**Comment by alcaeus** [comment]
Ah, looks like GitHub doesn't notify on a new tag, only on a new release. Thanks!

**Pull request merged**